components/endpoint-uptime-tracker/uptime-tracker.tsx

```
@@ -2,13 +2,16 @@ import { memo } from 'react'
 
 import { getHourAlignedTimestamp } from '@/convex/shared'
 
-import { useEndpointUptimes } from '@/hooks/api'
 import { formatTimestampToYMDHM } from '@/lib/utils'
 
 import { Tracker } from './tracker'
 
-function UptimeTracker_({ endpoint_uuid }: { endpoint_uuid: string }) {
-  const uptimeMetrics = useEndpointUptimes(endpoint_uuid)
+function UptimeTracker_({}: { endpoint_uuid: string }) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)